lms/djangoapps/grades/subsection_grade_factory.py

```diff
@@ -41,26 +40,25 @@ def create(self, subsection, read_only=False, force_calculate=False):
 
         If read_only is True, doesn't save any updates to the grades.
         force_calculate - If true, will cause this function to return a `CreateSubsectionGrade` object if no cached
-        grade currently exists, even if the assume_zero_if_absent flag is enabled for the course.
+        grade currently exists.
         """
         self._log_event(
             log.debug, f"create, read_only: {read_only}, subsection: {subsection.location}", subsection,
         )
 
         subsection_grade = self._get_bulk_cached_grade(subsection)
         if not subsection_grade:
-            if assume_zero_if_absent(self.course_data.course_key) and not force_calculate:
```

Comment:
now `assume_zero_if_absent` is False by default could you confirm that the current condition works as aspected considering that first condition is False?